src/current/v25.1/check-external-connection.md

```diff
@@ -0,0 +1,93 @@
+---
+title: CHECK EXTERNAL CONNECTION
+summary: Test the connection of each node to your cloud storage location.
+toc: true
+---
+
+{% include_cached new-in.html version="v25.1" %} The `CHECK EXTERNAL CONNECTION` tests the connection from each node in the cluster to an external cloud storage location. `CHECK EXTERNAL CONNECTION` will measure the time it takes each node to write a file, read it, and delete it from the specified storage location. You can adjust the number and concurrency of the test runs as well as the size of the file to write and read for each test.
+
+{{site.data.alerts.callout_info}}
+You can use the `CHECK EXTERNAL CONNECTION` to test the connection to [**cloud storage**]({% link {{ page.version.version }}/use-cloud-storage.md %}) locations. 
+{{site.data.alerts.end}}
+
+## Synopsis
+
+<div>
+{% remote_include https://raw.githubusercontent.com/cockroachdb/generated-diagrams/{{ page.release_info.crdb_branch_name }}/grammar_svg/check_external_connection.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------+------------
+`connection_uri` | The URI to the external storage. Specify the [provider's URI]({% link {{ page.version.version }}/use-cloud-storage.md %}) (e.g., `gs://bucket_name?AUTH...`) or a user-defined [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) (e.g., `external://gcs`).
+
+## Options
+
+Option  | Value | Description
+--------+-------+------------
+`concurrently` | `INT` | Run multiple connection tests concurrently. If you also set the `time` option, it will run the specified number of concurrent tests until the time has elapsed. By default, only `1` connection test will run.
```

Comment:
i assume `concurrentLY` is intentional and is not `concurrenCY` ?

Reply:
Yep, it's `concurrently`!